Specs/SpecRunner.html

```diff
@@ -21,14 +21,17 @@
         if (window.location.search.indexOf('built') !== -1) {
             if (window.location.search.indexOf('release') !== -1) {
                 document.write('<script src="../Build/Cesium/Cesium.js"><\/script>');
+                document.write('<link rel="stylesheet" href="../Build/Cesium/Widgets/CesiumWidget/CesiumWidget.css">');
```

Comment:
There's a top-level widgets.css that imports all widget styles, probably better to use that throughout.